client/src/components/Sidebar/Contacts/Contacts.js

```diff
@@ -12,12 +12,18 @@ const Contacts = props => {
   const toggleDisplay = (event, setting) => {
     setDisplay(setting);
   }
+ const [friends, setFriends] = React.useState([]);
+ const email = localStorage.getItem('email'); 
+ const loadFriends = async() => {
+   const res = await axios.get(`http://localhost:3001/invitations/user/${email}/contacts`);
```

Comment:
has this route been created? if so can you lead me to the backend file? 

we probably don't want to pass in email to the backend, rather than the id

Reply:
@bonnieli I made that change after the invitations model was updated to store to and from emails. it seemed strange to include the email in the url but I also tried passing it in the query body. I guess I used the email so fewer queries were needed. but it can be changed back.